### PR TITLE
DDF-4073 Document Intrigue metadata upload limitations

### PR DESCRIPTION
**Pull request by ethantmanns**
#### What does this PR do?
Adds a warning in the docs to warn users that Intrigue uploads will be considered as products instead of metadata.

#### Who is reviewing it? 

@jhunzik 
@paouelle 
@tyler30clemens 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams/docs
#### Ask 2 committers to review/merge the PR and tag them here.
@figliold
@ricklarsen - Documentation
@tbatie
@shaundmorris 

#### How should this be tested?

Build and view docs.

#### What are the relevant tickets?
[DDF-4073](https://codice.atlassian.net/browse/DDF-4073)

#### Checklist:
- [X] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by tyler30clemens** [comment]
❓ "If the correct DDMS transformer is tried first, it will work and the result will be a ddms20.metacard or ddms50.metacard, but if the XML Transformer is tried first, the result will be a ?? metacard type." -- is this supposed to be explicitly mentioned anywhere here? Or is this assumed/unnecessary/too much info?

File: distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc [hunk 1]
```diff
@@ -111,6 +111,12 @@ Workspaces can be shared between users at different levels of access as needed.
 
 Data can be ingested via ${catalog-ui}.
 
+[NOTE]
```
[the remark this answers is not in the document]
Comment (ethantmanns) [reply distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc]: I thought about that a lot but didn't wasn't sure what was more appropriate. If you have a preference to warning I can change it.

[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc]: I'll vote for warning

**Comment by ricklarsen** [comment]
I wonder if it'd be helpful to add some clarification about the difference between products and metadata files. I don't believe we define either anywhere. Could be follow-on work, maybe?

**Comment by ethantmanns** [comment]
@ricklarsen Maybe some quick examples? 
`The Intrigue uploader is intended for the upload of products (such as images or documents), not metadata files (such as Metacard XML).`

**Comment by ethantmanns** [comment]
build now

**Comment by ethantmanns** [comment]
build now

**Comment by ethantmanns** [comment]
build now

**Comment by rzwiefel** [comment]
@ethantmanns I think we just need someone to build+view docs and we can :shipit:

**Comment by tyler30clemens** [comment]
Hero'd Successfully 
built and verified changes were there

**Comment by ethantmanns** [comment]
build now

**Pull request merged**